openslides_backend/models/fields.py

```diff
@@ -7,6 +8,12 @@
 Schema = Dict[str, Any]
 
 
+class OnDelete(Enum):
+    PROTECT = "PROTECT"
+    CASCADE = "CASCADE"
+    SET_NULL = "SET_NULL"
```

Comment:
This should be renamed to something like "ENABLED" or "PASS" or "RESET" because it says that the related object stays as is and only the relation is reset. It is not always a set to null because we can have a list at the relation side where we just remove the respective instance id.

Reply:
I think the names are still fitting.

- `PROTECT` means: _protect_ this model as long as this field is not empty
- `CASCADE` means: if this model gets deleted, _cascade_ this deletion to the objects noted in this field
- `SET_NULL` is the only ambigous name since it can also mean "delete from array", but I don't have an idea for a better name here. However, since it's the default, it shouldn't be noted anywhere in the `models.yml`, so it should be no problem to change it later if we come up with a better name.